stdlib/pyclbr.pyi

```diff
@@ -1,19 +1,30 @@
 import sys
-from collections.abc import Sequence
+from collections.abc import Mapping, Sequence
 
 __all__ = ["readmodule", "readmodule_ex", "Class", "Function"]
 
-class Class:
+class _Object:
     module: str
     name: str
-    super: list[Class | str] | None
-    methods: dict[str, int]
     file: int
     lineno: int
 
     if sys.version_info >= (3, 10):
         end_lineno: int | None
 
+    parent: _Object | None
+    children: Mapping[str, _Object]
+
+    if sys.version_info >= (3, 10):
+        def __init__(
+            self, module: str, name: str, file: str, lineno: int, end_lineno: int | None, parent: _Object | None
+        ) -> None: ...
+    else:
+        def __init__(self, module: str, name: str, file: str, lineno: int, parent: _Object | None) -> None: ...
+
+class Class(_Object):
+    super: list[Class | str] | None
```

Comment:
I know that you only moved it here: But this will have similar issues like `children` above, especially in the constructor. (I.e. `super = ["foo"]; Class(..., super, ...)` will fail.) We should probably use `Sequence` here and in the constructor, but no need to fix this in this PR, as it hasn't made problems so far.